src/components/artistas/ArtistaPortfolio.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { toastr } from 'react-redux-toastr';
+
+import Uploads from '../fields/Uploads';
+import { ToastrOptionsConfirm } from '../../utilities/toastr.js';
```

Comment:
we should use this utility more as a good overlay

Reply:
I think we have basic js confirms somewhere that should be replaced

Reply:
I think they all got changed already :)